src/app/result/[postId]/_components/VoteProgress.tsx

```diff
@@ -0,0 +1,39 @@
+interface Option {
+  seq: number;
+  content: string;
+  count: number;
+}
+
+interface VoteProgressProps {
+  title: string;
+  options: Option[];
+  totalNumber: number;
+}
+
+const VoteProgress = ({ title, options, totalNumber }: VoteProgressProps) => {
+  const percentageOf = (count: number) => {
```

Comment:
```suggestion
  const percentageOf = (count: number) => ((count / totalNumber) * 100)
```

다음과 같이수정할 수도 있겠네요 ! resolve는 확인하고 수혁님이 해주세요 :) 로직상의 문제는 아니라서요 ! 고생하셨습니다.